src/config/dynamic-theme-fixes.config

```diff
@@ -17888,6 +17891,15 @@ nav a[href="/"] svg > g > path:first-child
 
 ================================
 
+urpredditodicittadinanza.lavoro.gov.it
+
+CSS
+:root{
+--lwc-colorContentAreaBackground : rgba(36, 39, 41, 0.98);
```

Comment:
Please use here as well the `${...}` form.

Reply:
ok, i've change it